src/KeyBindingsManager.ts

```diff
@@ -0,0 +1,264 @@
+/*
+Copyright 2021 Clemens Zeidler
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { defaultBindingsProvider } from './KeyBindingsDefaults';
+import { isMac } from './Keyboard';
+
+/** Actions for the chat message composer component */
```

Comment:
For single line comments, we mostly use `//` style, so perhaps best to use that for all these...?

Reply:
I used `/** */` because VSCode will show the docs in the editor, e.g. when hovering over the enum in the code. I find this quite handy when reading the code since you don't have to jump to the definition to get the docs.

What do you prefer? Using multi-line comments
```
/**
 * Description...
 */
 ```
or just `// Description`?

Reply:
Just `// Description`

Reply:
Was just think about why VSCode / the TS language server is doing it. Most likely it's because I am using JSDoc/TSDoc style comments to document the enum (and I think thats a good think to do here). However, putting it in one line is probably not that common.

Reply:
Hmm, okay, I suppose then the one line `/** */` style has more value here than I thought, so it seems fine to keep it.

In that case, please check all your comments, as some have only one initial star instead of two, and some are missing starting / ending spaces.